spaceship/docs/iTunesConnect.md

```diff
@@ -296,22 +296,30 @@ app.add_external_tester!(email: "github@krausefx.com", first_name: "Felix", last
 Right now, `spaceship` can't modify or create internal testers.
 
 # Load all sandbox testers
+```ruby
 testers = Spaceship::Tunes::SandboxTester.all
+```
 
 # Create a sandbox tester
```

Comment:
Good catch.

I left them as code comments, they looks members of the same section "Testers" in the document.
https://github.com/fastlane/fastlane/pull/7002

Thanks